ui/src/components/Exchange/Exchange.tsx

```diff
@@ -97,6 +98,7 @@ const Exchange: React.FC<Props> = ({ onLogout }) => {
                                  items={[
                                     {to: `${url}/market-pairs`, label: 'Market Pairs', icon: <PublicIcon/>},
                                     {to: `${url}/participants`, label: 'Investors', icon: <UserIcon/>}
+                                    // {to: `${url}/derivatives`, label: 'Derivatives', icon: <PublicIcon/>},
```

Comment:
is this ready?

Reply:
it technically works, but it is basically a stub that just has a list with the names. I figured we'd add it in once that page is finished, or make an "Assets" page with both tokens and derivatives that everyone can see